content/chainguard/chainguard-images/getting-started/cilium/index.md

```diff
@@ -102,7 +103,7 @@ export OPERATOR_IMAGE=cgr.dev/$ORGANIZATION/cilium-operator-generic:latest
 
 > **Note**: If you don't remember the name of your Chainguard organization, you can find it by running `chainctl iam organizations list -o table`.
 
-After that, you can install Cilium using the following command:
+After that, you install Cilium using the following command:
```

Comment:
maybe delete "you" as well? (I think this is very subjective, so feel free to ignore).